src/TodoList.jsx

```diff
@@ -1,31 +1,48 @@
 import PropTypes from "prop-types";
 import AddTodoForm from "./AddTodoForm";
 
-const todoList = [
-  { id: 1, title: "Add new task" },
-  { id: 2, title: "Edit" },
-  { id: 3, title: "Task done" },
-];
+TodoList.propTypes = {
+  onAddTodo: PropTypes.func.isRequired,
+  todoList: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
+      title: PropTypes.string.isRequired,
+    })
+  ).isRequired,
+};
+UnorderedList.propTypes = {
+  todoList: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
+      title: PropTypes.string.isRequired,
+    })
+  ).isRequired,
+};
+
+AddTodoForm.propTypes = {
+  onAddTodo: PropTypes.func.isRequired,
+};
```

Comment:
The `propTypes` of the `AddTodoForm` component are already defined in its own file, so you can remove this definition from here.

Reply:
I have tried, but it shows me mistake, and ask for validation. In addition to go without validation I pass into ESlint rules: {
      "react/prop-types": "off", my fellow students use such a rule and void prop validation, but in my cases it still asks